packages/next/src/views/Login/LoginForm/index.tsx

```diff
@@ -80,6 +81,7 @@ export const LoginForm: React.FC<{
       disableSuccessStatus
       initialState={initialState}
       method="POST"
+      onSuccess={fetchFullUser}
```

Comment:
We should not have to fetch the full user after log in because the login response itself contains the user.